README.md

```diff
@@ -97,67 +113,93 @@ datadog.addLambdaFunctions([<LAMBDA_FUNCTIONS>])
 datadog.addForwarderToNonLambdaLogGroups([<LOG_GROUPS>])
 ```
 
+#### Go
+
+```go
+import (
+	"github.com/DataDog/datadog-cdk-constructs/dist/go/ddcdkconstruct"
+)
+datadog := ddcdkconstruct.NewDatadog(
+    stack,
+    jsii.String("Datadog"),
+    &ddcdkconstruct.DatadogProps{
+        NodeLayerVersion:      jsii.Number(<LAYER_VERSION>),
+        AddLayers:             jsii.Bool(<BOOLEAN>),
+        Site:                  jsii.String(<SITE>),
+        ApiKey:                jsii.String(os.Getenv("DD_API_KEY")),
+        // ...
+    })
+    datadog.AddLambdaFunctions(&[]interface{}{myFunction}, nil)
+    datadog.AddForwarderToNonLambdaLogGroups()
+```
+
 ## Source Code Integration
+
 [Source code integration](https://docs.datadoghq.com/integrations/guide/source-code-integration/) is enabled by default through automatic lambda tagging, and will work if:
 
-- The Datadog Github Integration is installed.
+- The Datadog Github integration is installed.
 - Your datadog-cdk dependency satisfies either of the below versions:
   - `datadog-cdk-constructs-v2` >= 1.4.0
   - `datadog-cdk-constructs` >= 0.8.5
 
 ### Alternative Methods to Enable Source Code Integration
-If the automatic implementation doesn't work for your case, please follow one of the two guides below. 
+
+If the automatic implementation doesn't work for your case, please follow one of the two guides below.
 
 **Note: these alternate guides only work for Typescript.**
+
 <details>
-  <summary>datadog-cdk version satisfied, but Datadog Github Integration NOT installed</summary>
+  <summary>datadog-cdk version satisfied, but Datadog Github integration NOT installed</summary>
 
-  If the Datadog Github Integration is not installed, you need to import the `datadog-ci` package and manually upload your Git metadata to Datadog.
-  We recommend you do this where your CDK Stack is initialized.
+If the Datadog Github integration is not installed, you need to import the `datadog-ci` package and manually upload your Git metadata to Datadog.
+For the best results, import the `datadog-ci` package where your CDK Stack is initialized.
```

Comment:
Addressing the comment from documentation team
https://github.com/DataDog/datadog-cdk-constructs/pull/271/files/6a32a452d56f58ff6f6cf936ff9d7bd0449fe02c#r1702366653